src/kontrol/options.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+
+    from pyk.utils import BugReport
+
+
+@dataclass(frozen=True)
+class ProveOptions:
+    auto_abstract_gas: bool
+    bug_report: BugReport | None
+    use_booster: bool
+    kore_rpc_command: str | Iterable[str] | None
```

Comment:
Now that you have an `__init__`, you can strengthen the types, e.g.:

```suggestion
    kore_rpc_command: tuple[str, ...]
```

Reply:
Fixed.